runtime/kusama/src/lib.rs

```diff
@@ -584,7 +555,7 @@ construct_runtime! {
 
 		// Consensus support.
 		Authorship: authorship::{Module, Call, Storage},
-		Staking: staking::{default, OfflineWorker},
```

Comment:
@bkchr `OfflineWorker` is no longer supported?

Reply:
That is okay. This was never a valid syntax. I'm not even sure what `OfflineWorker` should refer too. If that should refer to offchain workers, they are implemented like `OnInitialize` etc.